### PR TITLE
DevEx: CI Docker image to ECR

### PR DESCRIPTION
**Pull request by sutt0n**
- removed dockerhub image utilization from CircleCI config
- added docker image versioning `docker-to-ecr.sh`
- split up pa11y tests due to underlying memory problems with dependencies (puppeteer and its dependency, chrome)

### CODE REVIEW

**Comment by kkoskelin** [comment]
nitpick, but fyi puppeteer is actually using `chrome` (not chromium), hence the `puppeteer-core` which I believe drives the `chrome-aws-lambda` instance.

File: package.json [hunk 1]
```diff
@@ -195,7 +195,8 @@
     "test:client:integration": "npm run test:_client -- \"web-client/integration-tests/.*\\.test\\.js\"",
     "test:public:integration": "npm run test:_client -- \"web-client/integration-tests-public/.*\\.test\\.js\"",
     "test:client:unit": "npm run test:_client -- \"web-client/src/.*\\.test\\.js\"",
-    "test:pa11y": "pa11y-ci --config web-client/pa11y-ci.config.js",
```
Comment (matthopson) [inline package.json:198]: Do you think we should leave this command and just chain 1 && 2?

File: web-api/serverless-users.yml [hunk 2]
```diff
@@ -156,7 +156,10 @@ functions:
       - http:
           path: /
           method: get
-          cors: ui-${self:provider.stage}.ef-cms.${opt:domain}
+          cors: 
+            origins:
+              - http://localhost:1234
```
Comment (matthopson) [inline web-api/serverless-users.yml]: Is this necessary?

File: web-client/pa11y-ci.web-client-1.config.js [hunk 3]
```diff
@@ -2,19 +2,8 @@ const adc = require('./pa11y/pa11y-adc');
 const docketclerk = require('./pa11y/pa11y-docketclerk');
 const judge = require('./pa11y/pa11y-judge');
 const petitioner = require('./pa11y/pa11y-petitioner');
-const petitionsclerk = require('./pa11y/pa11y-petitionsclerk');
```
Comment (matthopson) [inline web-client/pa11y-ci.web-client-1.config.js]: Do we need to add these back?

**Review by matthopson — APPROVED** [review]
shipppppppppp

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**